views/components/sample_component/sample_component.hbs

```diff
@@ -1,4 +1,6 @@
-<div class="sample-component">
-  <h2 class="sample-component__heading">{{heading}}</h2>
-  <p>{{body}}</p>
+<div class="grid__cell">
```

Comment:
I think if possible the markup for a component should exist independently of whatever is going to contain it. So e.g. the grid cell would live with the grid markup, and the components would get dropped into the grid. Admittedly at the expense of having a bit more markup in `layouts.hbs`. I suppose we could have a `renderComponentInGridCell` (or similarly-catchily-titled) helper that does the wrapping. Reckons?